testing/tests/DevExpress.ui.widgets.scheduler/layoutManager.tests.js

```diff
@@ -639,8 +650,60 @@ QUnit.test("Grouped appointments schould have correct colors", function(assert)
     var $appointment = $(this.instance.$element().find(".dx-scheduler-appointment"));
     assert.equal($appointment.length, 2, "Cloned appointments are grouped");
 
-    var $dropDownMenu = $(this.instance.$element()).find(".dx-scheduler-dropdown-appointments").trigger("dxclick");
+    var $dropDownMenu = $(this.instance.$element()).find(".dx-scheduler-dropdown-appointments");
+
+    assert.equal(new Color($dropDownMenu.css("background-color")).toHex(), "#0000ff", "ddAppointment is rendered");
+});
+
+QUnit.test("Grouped appointments schould have correct colors when resourses store is asynchronous", function(assert) {
+    var items = [], i = 2;
+
+    while(i > 0) {
+        items.push({ text: i, startDate: new Date(2015, 1, 9), endDate: new Date(2015, 1, 9, 1), roomId: 1 });
+        i--;
+    }
+    i = 10;
+    while(i > 0) {
+        items.push({ text: i, startDate: new Date(2015, 1, 9), endDate: new Date(2015, 1, 9, 1), roomId: 2 });
+        i--;
+    }
+
+    this.createInstance(
+        {
+            currentDate: new Date(2015, 1, 9),
+            currentView: "month",
+            height: 500,
+            dataSource: items,
+            resources: [
+                {
+                    field: "roomId",
+                    allowMultiple: true,
+                    dataSource: new DataSource({
```

Comment:
`DataSource` wrapper isn't required here. You can use `CustomStore` as a scheduler data source.